### PR TITLE
MOTS-139: Add possibility to edit modules

### PR DESCRIPTION
**Pull request by pmuchowski**

### CODE REVIEW

File: src/webapp/js/components/modules-manage.js [hunk 1]
```diff
@@ -22,6 +64,73 @@ export default class ModulesManage extends Component {
     }
   }
 
+  getNodeClassName(node) {
+    const classNames = [];
+
+    if (node.type !== 'MODULE') {
```
Comment (awalkowiak) [inline src/webapp/js/components/modules-manage.js]: Would not be more convinient here to have dict with key - node.type and value className?
eg.:
const nodeTypes = {
 nodeType1: 'class-name-1',
 nodeType2: 'class-name-2',
};
classNames.push(nodeTypes[node.type]);

Comment (pmuchowski) [reply src/webapp/js/components/modules-manage.js]: it's just one if, rest of the node type is not checked, they all have the same class

Comment (awalkowiak) [reply src/webapp/js/components/modules-manage.js]: ok

**Comment by awalkowiak** [comment]
lgtm

**Pull request merged**